README.md

```diff
@@ -1,48 +1,48 @@
 # dags
 
-This is a [Dagster](https://dagster.io/) project scaffolded with [`dagster project scaffold`](https://docs.dagster.io/getting-started/create-new-project).
+Dagster project containing code for updating, managing, and viewing OCF's server-stored datasets.
 
-## Getting started
+## Ubiquitous language
 
-First, install your Dagster code location as a Python package. By using the --editable flag, pip will install your Python package in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs) so that as you develop, local code changes will automatically apply.
-
-```bash
-pip install -e ".[dev]"
-```
+The following terms are used throughout the codebase and documentation. They are defined here to avoid ambiguity.
 
-Then, start the Dagster UI web server:
+ - *InitTime* - The time at which a forecast is initialised. For example, a forecast initialised at 12:00 on 1st January.
+ - *TargetTime* - The time at which a predicted value is valid. For example, a forecast with InitTime 12:00 on 1st January predicts that the temperature at TargetTime 12:00 on 2nd January at position x will be 10 degrees.
 
-```bash
-dagster dev
-```
+## Conventions
 
-Open http://localhost:3000 with your browser to see the project.
+The storage of data is handled automatically into locations defined by features of the data in question. The only configurable
+part of the storage is the *Base Path* - the root point from which dagster will then handle the subpaths. These paths are generated
+via the following features:
+ - The *flavour* of the data (NWP, Satellite etc)
```

Comment:
Very minor nitpick, and maybe we should change it, but we do officially use American English in our [code style](https://github.com/openclimatefix/.github/blob/main/coding_style.md)

```suggestion
 - The *flavor* of the data (NWP, Satellite etc)
```